### PR TITLE
RDM-5586: Publish Annotation APIs From CCD Gateway

### PR DESCRIPTION
**Pull request by MKaramath**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-5586

### Change description ###
Media viewer - Updated to include the annotations API

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -94,6 +96,13 @@ applyProxy(app, {
   rewrite: false
 });
 
+app.use('/em-anno', proxy({
```
Comment (andrewfolga) [inline app.js]: Can we use applyProxy here to be consistent with other usages?

File: infrastructure/main.tf [hunk 2]
```diff
@@ -103,6 +104,7 @@ module "api-gateway-web" {
     PROXY_DATA = "http://ccd-data-store-api-${local.env_ase_url}"
     PROXY_DEFINITION_IMPORT = "http://ccd-definition-store-api-${local.env_ase_url}"
     PROXY_DOCUMENT_MANAGEMENT = "${local.document_management_url}"
+    PROXY_MV_ANNOTATIONS_API_URL = "${local.mv_annotations_api_url}"
```
Comment (mario-paniccia) [inline infrastructure/main.tf]: is the declaration of a local var needed? 
Can you try removing line 22 and here directly with:
```
PROXY_MV_ANNOTATIONS_API_URL = "${var.mv_annotations_api_url}"
```

Comment (MSancaktutar) [reply infrastructure/main.tf]: Fixed.

File: app.js (hunk 1 above)
Comment (MKaramath) [reply app.js]: Andrew - As discussed, due to the pathRewrite value, I am using app.use function as is to avoid any further complexities.

Comment (andrewfolga) [reply app.js]: As discussed it is up to you but I see it as a quick win. Here are some details which I already tested locally and worked fine:
```
  if (false !== config.rewrite) {
    options.pathRewrite = {
        [`^${config.source}`]: config.rewriteUrl || ''
    };
  }
```
and then:
```
applyProxy(app, {
  source: '/em-anno',
  target: config.get('proxy.mv_annotations'),
  rewrite: false,
  rewriteUrl: '/api'
});
```

Comment (MSancaktutar) [reply app.js]: done

File: config/default.yaml [hunk 3]
```diff
@@ -2,18 +2,19 @@ proxy:
   aggregated: http://localhost:4452
   data: http://localhost:4452
   definition_import: http://localhost:4451
-  document_management: http://localhost:4603
+  document_management: http://localhost:4506
```
Comment (mario-paniccia) [inline config/default.yaml]: for my knowledge, how come these changes in port the numbers in this file?

Comment (MSancaktutar) [reply config/default.yaml]: Well, they should have been fixed before, to run the gateway outside a docker container. We had to fix these to make it work.

**Pull request merged**